src/code.rs

```diff
@@ -589,6 +651,33 @@ fn build_table_fns(
 
     buffer.push_str("}\n");
 
+    #[cfg(feature = "advanced-queries")]
+    // generate filter struct for filter() helper function
+    {
+        let filter_fields = table
+            .columns
+            .iter()
+            .map(|column| {
+                let struct_field = StructField::from(column);
+                format!(
+                    "pub {column_name}: Option<{column_type}>,",
+                    column_name = struct_field.name,
+                    column_type = struct_field.to_rust_type()
+                )
+            })
+            .collect::<Vec<_>>()
+            .join("\n    ");
+
+        buffer.push_str(&format!(
```

Comment:
```suggestion
        buffer.push_str(&formatdoc!(
```
it is more indented than it should be (see generated code)